relevanceai/http_client.py

```diff
@@ -135,3 +141,16 @@ def make_search_suggestion(self):
 
     def check_auth(self):
         return self.admin._ping()
+
+    def check_package_version(self):
```

Comment:
This isn't the right place to put it. This needs to run on import step. 

1) move this to a `utils.py` 
2) Move this such that this should run when people run 
```
from relevanceai import Client

# outputs warning of not being the latest version 
```